### PR TITLE
[CodeCamp #1536]  Adopt mmeval's MattingMSE in mmediting

### PR DESCRIPTION
**Pull request by xuan07472**
## Motivation

Adopt mmeval's MattingMSE in mmediting

## Modification

Modify the matting_mse.py file to support mmeval.

## Evaluation test ([dev-1.x](https://github.com/open-mmlab/mmediting/tree/dev-1.x))
Time cost 18:02 (from log)
[20230109_162602.log](https://github.com/open-mmlab/mmediting/files/10371669/20230109_162602.log)
image

## Evaluation test (this PR)
Time cost 16:03 (from log)
[20230109_160805.log](https://github.com/open-mmlab/mmediting/files/10371666/20230109_160805.log)
image

### CODE REVIEW

File: mmedit/evaluation/metrics/matting_mse.py [hunk 1]
```diff
@@ -55,18 +51,15 @@ def process(self, data_batch: Sequence[dict],
...
-            weight_sum = (trimap == 128).sum()
-            if weight_sum != 0:
-                mse_result = ((pred_alpha - gt_alpha)**2).sum() / weight_sum
-            else:
-                mse_result = 0
-
-            self.results.append({'mse': mse_result})
-
-    def compute_metrics(self, results: List):
+    def compute_metrics(self, results: List) -> Dict[str, float]:
```
Comment (ice-tong) [inline mmedit/evaluation/metrics/matting_mse.py:62]: The `compute_metrics` could be remove~

File: mmedit/evaluation/metrics/matting_mse.py [hunk 2]
```diff
@@ -1,19 +1,20 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-from typing import List, Sequence
+from typing import Dict, List, Optional, Sequence
 
-from mmengine.evaluator import BaseMetric
+from mmeval import MattingMSE as MattingMSE_MMEVAL
```
Comment (ice-tong) [inline mmedit/evaluation/metrics/matting_mse.py:4]:
```suggestion
from mmeval import MattingMSE as _MattingMSE
```

File: mmedit/evaluation/metrics/matting_mse.py [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
 
 from mmedit.registry import METRICS
-from .metrics_utils import _fetch_data_and_check, average
+from .metrics_utils import _fetch_data_and_check
 
 
 @METRICS.register_module()
-class MattingMSE(BaseMetric):
-    """Mean Squared Error metric for image matting.
+class MattingMSE(MattingMSE_MMEVAL):
```
Comment (ice-tong) [inline mmedit/evaluation/metrics/matting_mse.py:11]:
```suggestion
class MattingMSE(_MattingMSE):
```

**Comment by ice-tong** [comment]
Please add the evaluation result before and after this PR~

File: mmedit/evaluation/metrics/matting_mse.py (hunk 1 above)
Comment (xuan07472) [reply mmedit/evaluation/metrics/matting_mse.py]: Removed

**Pull request closed without merging**